rc/tools/spell.kak

```diff
@@ -148,3 +148,12 @@ define-command \
     ')
     printf 'try %%{ menu -auto-single %s }' "${menu}"
 } }
+
+define-command \
+    -docstring "Add the current selection to the dictionary" \
+    spell-add %{ nop %sh{
```

Comment:
Any way you could make the function 1) use the word passed as argument, if specified 2) or otherwise add all the words in the current multiple selection?